erica_app/erica/routes.py

```diff
@@ -10,10 +10,10 @@
     UnlockCodeActivationRequestController, EstValidationRequestController, EstRequestController, \
     UnlockCodeRevocationRequestController
 
-PYERIC_VERSION_URL = '/01'
+ERICA_VERSION_URL = '/01'
```

Comment:
I would propose to update this version either now or when we add a new route for getting the list of the tax offices. WDYT?

Reply:
Hm, I disagree. I would suggest we upgrade the version no sooner than when the following criteria are met:
- we introduce a breaking change
- we have either have clients we do not control (public or other teams) OR we cannot upgrade all clients at the same time as we change Erica.

As it stands, we have a single client under our full control, which we can adapt in the same PR, so changing the version would introduce work with no gain.

Reply:
Okay, point taken.